app/components/forms/wizard/wizard-footer.hbs

```diff
@@ -33,7 +33,7 @@
   </div>
 </div>
 <div class="ui grid">
-  <div class="sixteen wide column right aligned">
+  <div class="sixteen wide column left aligned">
```

Comment:
Why?

Reply:
Plz, [refer](https://github.com/fossasia/open-event-frontend/pull/6153#issuecomment-751460775) to this above. Thanks Sir.